cmd/client-s3.go

```diff
@@ -36,7 +36,7 @@ import (
 
 	"github.com/minio/mc/pkg/httptracer"
 	"github.com/minio/mc/pkg/probe"
-	"github.com/minio/minio-go"
+	minio "github.com/minio/minio-go"
```

Comment:
@poornas This code has not been used anywhere else in this file. Is this change required?

Reply:
This is perhaps automatic with `goimports` through the editor.